pages/contribute/working_with_git.md

```diff
@@ -60,7 +60,7 @@ To run the website locally, you can either use [Docker](https://www.docker.com/)
 
 1. If not already installed on your machine, install Docker. From the root of the ``rdmkit`` directory, run:
     ```
-    docker run -it --rm -p 4000:4000 -v $PWD:/srv/jekyll jekyll/jekyll:latest /bin/bash -c "gem install github-pages --no-document && chmod 777 /srv/jekyll && jekyll serve -w"
+    docker run -it --rm -p 4000:4000 -v $PWD:/srv/jekyll jekyll/jekyll:latest /bin/bash -c "chmod -R 777 /srv/jekyll && bundle install && bundle exec jekyll serve -w - --host 0.0.0.0 --livereload"   
```

Comment:
```suggestion
    docker run -it --rm -p 4000:4000 -v $PWD:/srv/jekyll jekyll/jekyll:3 /bin/bash -c "chmod -R 777 /srv/jekyll && bundle install && bundle exec jekyll serve -w - --host 0.0.0.0 --livereload"   
```

Using Jekyll 3 it works for me! Could it be that you guys had locally an older image? It would also explain why it used to work in the past for me. Pinning a version is a good habit anyway. With GitHub pages we are still at Jekyll 3 ...

Reply:
Yes sounds like a good idea to pin this to Jekyll 3 for now.